docs/www/quick-start-kubernetes.md

```diff
@@ -4,91 +4,178 @@ order: 0
\ No newline at end of file
+1. Using Helm, add the Redpanda chart repository and update it:
+
+    ```
+    helm repo add redpanda https://charts.vectorized.io/ && helm repo update
+    ```
+
+2. Install the Redpanda operator CRDs:
+
+    ```
+    kubectl apply -k 'https://github.com/vectorizedio/redpanda/src/go/k8s/config/crd?ref=<latest version>'
+    ```
+
+    You can find the latest version number of the operator on the [list of operator releases](https://github.com/vectorizedio/redpanda/releases).
+
+3. Install Redpanda on your Kubernetes cluster with:
+
+    ```
+    helm install --namespace redpanda-system --create-namespace redpanda-operator redpanda/redpanda-operator
+    ```
+
+## Next steps
+
+After you set up Redpanda in your Kubernetes cluster, you can:
+
+- Install resources from our sample files:
+    - One node cluster - 
+                
+        ```
+        kubectl apply -n redpanda-system -f https://raw.githubusercontent.com/vectorizedio/redpanda/dev/src/go/k8s/config/samples/one_node_cluster.yaml
+        ```
+
+    - Cluster with external connectivity (verified on AWS) -
+        
+        ```
+        kubectl apply -n redpanda-system -f https://raw.githubusercontent.com/vectorizedio/redpanda/dev/src/go/k8s/config/samples/external_connectivity.yaml
+        ```
+
+    - Cluster with TLS encryption -
+        
+        ```
+        kubectl apply -n redpanda-system -f https://raw.githubusercontent.com/vectorizedio/redpanda/dev/src/go/k8s/config/samples/redpanda_v1alpha1_with_tls.yaml)
+        ```
+
+        To use the TLS encryption cluster, you have to mount a TLS configuration for rpk.
+        The following definition creates a pod that has TLS configured.
+        From that pod, users can list, create, and delete topics, and also produce and consume events.
+
+        ```
+        cat <<EOF | kubectl apply -f -
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+         name: rpk-config
+        data:
+         redpanda.yaml: |
+           redpanda:
+           rpk:
+             tls:
+               truststore_file: /etc/tls/certs/ca.crt
+        ---
+        apiVersion: v1
+        kind: Pod
+        metadata:
+         name: produce-msg
+        spec:
+         volumes:
+         - name: tlscert
+           secret:
+             defaultMode: 420
+             secretName: cluster-sample-tls-redpanda
+         - name: rpkconfig
+           configMap:
+             name: rpk-config
+         containers:
+         - name: rpk
+           image: vectorized/redpanda:latest
+           command:
+           - /bin/bash
+           tty: true
+           stdin: true
+           volumeMounts:
+           - mountPath: /etc/tls/certs
+             name: tlscert
+           - mountPath: /etc/redpanda
+             name: rpkconfig
+        EOF
+        ```
+
+        To connect to the produce pod, run:
+        
+        ```
+        kubectl attach -n redpanda-system -ti produce-msg
```

Comment:
```suggestion
        kubectl attach -n redpanda-test -ti produce-msg
```


or any other namespace name

Reply:
Do we have reference docs for rpk commands usage? 

It would be good to not left end-user in a state where they will have only command prompt. 

The desired set of command line one liners for tls example:

- `rpk topic create test-topic-name --brokers cluster-sample-tls-0.cluster-sample-tls.redpanda-test.svc.cluster.local:9092`
- `rpk topic list --brokers cluster-sample-tls-0.cluster-sample-tls.redpanda-test.svc.cluster.local:9092`
- `echo {"test":"message"} | rpk topic produce test-topic-name --brokers cluster-sample-tls-0.cluster-sample-tls.redpanda-test.svc.cluster.local:9092`
- `rpk topic consume test-topic-name --brokers cluster-sample-tls-0.cluster-sample-tls.redpanda-test.svc.cluster.local:9092`

Reply:
that's a great idea!!